fiona/collection.py

```diff
@@ -221,6 +221,87 @@ def crs_wkt(self):
             self._crs_wkt = self.session.get_crs_wkt()
         return self._crs_wkt
 
+    def tags(self, ns=None):
```

Comment:
If rasterio had not already set a precedent I would have suggested using `namespace=` instead of `ns=`.

Reply:
I had the same thought as I was going through this. Do you think it would be terrible to use `namespace=` instead?

Reply:
"ns" is not uncommon shorthand for namespace in the XML domain. "xmlns" for example, or the "start-ns" and "end-ns" events in the ElementTree API. I don't think there's any harm in using "ns" here as in Rasterio.